scipy/signal/signaltools.py

```diff
@@ -100,6 +100,19 @@ def correlate(in1, in2, mode='full'):
         ``same``
            The output is the same size as `in1`, centered
            with respect to the 'full' output.
+    method : str {'direct', 'auto', 'fft'}, optional
```

Comment:
The default option should be listed first  https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#user-content-sections